app/models/form_profile.rb

```diff
@@ -329,23 +329,27 @@ def vets360_contact_info_hash
     return_val
   end
 
+  def pciu_disabled?
+    Flipper.enabled?(:remove_pciu, user)
+  end
+
   def initialize_contact_information
     opt = {}
     opt.merge!(vets360_contact_info_hash) if vet360_contact_info
-    if Flipper.enabled?(:remove_pciu, user)
+    if pciu_disabled?
       # Monitor logs to validate the presence of Contact Information V2 user data
       Rails.logger.info("VAProfile Contact Info: Address? #{opt[:address].present?},
         Email? #{opt[:email].present?}, Phone? #{opt[:home_phone].present?}")
+    else
+      # The following pciu lines need to removed when tearing down the EVSS PCIU service.
+      opt[:email] ||= extract_pciu_data(:pciu_email)
+      if opt[:home_phone].nil?
+        opt[:home_phone] = pciu_primary_phone
+        opt[:us_phone] = pciu_us_phone
+      end
     end
     opt[:address] ||= user_address_hash
```

Comment:
Line 351, `opt[:address] ||= user_address_hash`, `user_address_hash` is an MPI request, not a PCIU request. This line should still be executed when remove_pciu is enabled.